### PR TITLE
now computing stats on subwindows within focal

### PR DESCRIPTION
**Pull request by mufernando**

### CODE REVIEW

**Comment by mufernando** [comment]
@nspope

File: workflows/sweep_simulate.snake [hunk 1]
```diff
@@ -162,17 +163,17 @@ def compute_clr(ts_path, recmap, model, samples, sample_sets):
...
         ]
         subprocess.run(args, stdout=subprocess.DEVNULL)
         clr = np.loadtxt(clr_path, skiprows=1)
-        # TODO: argmax CLR -- do we want to just use val from center of window instead?
-        # things get wierd on boundaries of grid if we only include SNPs in focal window
-        out.append(np.max(clr[:,1])) 
-    return np.array(out)
+        out.append(clr[:,1])
+    # windows for the CLR -- note that we are going from 1-based positions to [,) intervals
+    windows = np.array([grid, grid+1]).T
```
Comment (nspope) [inline workflows/sweep_simulate.snake:175]: maybe we should define the CLR grid as midpoints of the windows?

**Review by nspope — APPROVED** [review]
Looks good! I think we can worry about porting stuff to scikit-allel when incorporating @andrewkern's diploshic pipeline.

Next step is a draft figure for the CLR stuff, I think -- I can work on that later this week.

We might also want to add a rule that plots the stat grids in each window for each sim, and dumps this as a png in each directory -- for quality control. But can do that later.

**Comment by mufernando** [comment]
I added the VCF output to this PR @andrewkern

**Comment by andrewkern** [comment]
looks like the conda requirements have been moved to a different file? can you put everything into just the `environment.yml` file?

**Comment by andrewkern** [comment]
running into an issue where i don't have the reqs to run the sweep stuff as part of the main `analysis2` env

**Comment by mufernando** [comment]
this is fully working now! we had to downgrade `protobuf` to stop the weird seg faults we were getting.

**Comment by andrewkern** [comment]
i believe that @dschride is currently trying to run this pipeline on the UNC cluster to see if it works on their scheduler.

@mufernando do you reckon we are ready to merge?

**Comment by mufernando** [comment]
if it works fine in @dschride's cluster I think we can safely merge this.

**Comment by dschride** [comment]
I'm having all kinds of issues trying to run jobs of any kind on our cluster today so it may be a while longer before I can confirm this for you...

**Comment by andrewkern** [comment]
@dschride was able to confirm that this workflow ran to completion on his cluster

**Comment by dschride** [comment]
Yay!

**Pull request merged**